lib/smart_proxy_salt/salt.rb

```diff
@@ -14,10 +14,21 @@ class Plugin < ::Proxy::Plugin
 
       default_settings :autosign_file      => '/etc/salt/autosign.conf',
                        :salt_command_user  => 'root',
-                       :use_api            => false
+                       :use_api            => false,
+                       :saltfile           => '/etc/foreman-proxy/settings.d/salt.saltfile'
 
-      http_rackup_path File.expand_path('salt_http_config.ru', File.expand_path('../', __FILE__))
-      https_rackup_path File.expand_path('salt_http_config.ru', File.expand_path('../', __FILE__))
+      http_rackup_path File.expand_path('salt_http_config.ru', __dir__)
+      https_rackup_path File.expand_path('salt_http_config.ru', __dir__)
```

Comment:
I didn't comment on it since it was unrelated, but you can use with 2.3+
```suggestion
      rackup_path File.expand_path('salt_http_config.ru', __dir__)
```